scripts/linters/general_purpose_linter.py

```diff
@@ -609,6 +630,7 @@ def __init__(self, files_to_lint, verbose_mode_enabled):
         os.environ['PATH'] = '%s/bin:' % common.NODE_PATH + os.environ['PATH']
 
         self.files_to_lint = files_to_lint
+        self.file_cache = file_cache
```

Comment:
Is this the same fileCache shared among all the linters?

Reply:
yes